k8s/core/configmap/configmap_lock_v2_test.go

```diff
@@ -476,6 +476,21 @@ func TestIsKeyLocked(t *testing.T) {
 	require.True(t, locked)
 	require.Equal(t, id1, owner)
 
+	// simulate restart and keep lock unexpired
+	cmIntf, err = New("px-configmaps-lock-is-key-locked-test", nil, testLockTimeout, testLockAttempts, testLockRefreshDuration, testLockTTL)
+	require.NoError(t, err, "Unexpected error on New")
+	cm = cmIntf.(*configMap)
+
+	// check if locked and owner
+	locked, owner, err = cm.IsKeyLocked(key, id1)
+	require.NoError(t, err)
+	require.False(t, locked)
+	require.Equal(t, id1, owner)
+	locked, owner, err = cm.IsKeyLocked(key, id2)
+	require.NoError(t, err)
+	require.True(t, locked)
+	require.Equal(t, id1, owner)
+
```

Comment:
Can you include UT output without the fix?

Reply:
Updated in the description